definition-map.js

```diff
@@ -140,6 +140,26 @@ const QUERIES = {
             isPreview: "Requests the preview URL to launch SCORM content. This requires the current user to be in admin or manager role.",
             type: "The type of the Topic Or Course."
         }
+    },
+    CourseById: {
+        metadescription: "Returns various information about the course and the objects that are contained inside of the course, such as the sections, lessons, and pages associated with the course.",
```

Comment:
Since the sentence starts w/ "Returns various information about the course", I think we can omit the ending "associated with the course" as it's redundant.